apps/filebrowser/src/filebrowser/api.py

```diff
@@ -211,6 +211,25 @@ def rmtree(request):
   return HttpResponse(status=200)
 
 
+@error_handler
+def get_trash_path(request):
+  path = _normalize_path(request.GET.get('path'))
+  response = {}
+
+  trash_path = request.fs.trash_path(path)
+  user_home_trash_path = request.fs.join(request.fs.current_trash_path(trash_path), request.user.get_home_directory().lstrip('/'))
+
+  if request.fs.isdir(user_home_trash_path):
+    response['trash_path'] = user_home_trash_path
+  elif request.fs.isdir(trash_path):
```

Comment:
When is this happening, and is this base then still the _user's_ trash path or some shared trash?

Reply:
I didn't fully understand your comment here.

Reply:
In what scenario is the elif condition tested? i.e. why are there 2 different possible paths for the trash?